### ISSUE TITLE
Convert URL paths from /iaf/ to /ff/ to reflect the new name

### ISSUE THREAD

**Issue by jkosternl** [frankframework/frankframework#5849]
Currently, the FrankFramework uses `/iaf/` references in the URL paths, pointing to the deprecated name: IAF.
Also the URLs should be updated, to reflect the name changes. 

1. This needs to be updated in the documentation as well as in the code. 
1. Also it need to be mentioned as breaking change, in the release notes, so end-users are aware of it. 
2. The standard deployment URL of de `test` project should be changed from `iaf-test` to `ff-test`.

**Comment by nielsm5** [comment]
Dit gaat heel veel kapot maken. De deployment URL hangt aan de module naam vast, dus die kan wel binnenkort overgezet worden.
Hier zitten overigens ook wat haken en ogen aan.

**Comment by jkosternl** [comment]
> Dit gaat heel veel kapot maken. 
Ja, het is ook maar de vraag of het dan de moeite waard is. Deze afkorting is niet heel in het oog springend. Dus kunnen het ook parkeren. Er zijn belangrijkere naamswijzigingen die we moeten doen.

**Comment by nielsm5** [comment]
Bij deze parkeer ik hem.
